torch/distributed/rpc/api.py

```diff
@@ -51,7 +51,7 @@ def wrapper(*args, **kwargs):
 def wait_all_workers():
     r"""
     Block until all local and remote RPC processes reach this method, and then
-    destroy local the RPC agent. Every RPC process must call this method before
+    destroy RRef and RPC handlers. Every RPC process must call this method before
```

Comment:
"RPC handlers" -> "Python RPC handlers"


RPC handlers is vague.

Reply:
This docstring is also in the wrong place now, will update

Reply:
> I actually don't think we need to mention that we destroy RPC handlers and RRef context, those are internal details that should be abstracted away from the user.

Agree, users don't need to know there is a RRef context and handler